### ISSUE TITLE
Class wc-order Should have get_shipping_email()

### ISSUE THREAD

**Issue by mjdewitt** [woothemes/woocommerce#36955]
**Prerequisites (mark completed items with an [x]):**
- [x ] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x ] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

**Issue Description:**
The  method get_shipping_email()  is not an available method in class wc-order.

I don't believe that this is an enhancement or feature request as shipping_email is part of the full shipping address (which includes shipping_phone that is handled by get_shipping_phone). I believe that this is an error of omission.

For Business-to-Business emails, full billing and shipping addresses are a necessity. I was surprised that I had to use a get_post_meta( $order->id, '_shipping_email', true ) to get around this omission.

Thanks

Mike

**Comment by kevinruscoe** [comment]
I don't think "billing_email" exists.

- If you add an order in the admin, and go to edit/create a billing address, the field doesn't exist. 
- Viewing a customer in the admin, the shipping address doesn't have a field.
- Looking through the entire codebase for '_shipping_email' yields no results.
- Looking through the wp_postmeta table for '_shipping_email' yields no results.

If I run `get_post_meta( $order->id, '_shipping_email', true )` I get an empty string.

There's a good chance that you have a plugin that is saving this value.

**Comment by mjdewitt** [comment]
Hi,

My apologies. You are right, my addresses are fully fielded and had assumed that this had been addressed in prior updates. 

And so, I wrote a poem that I hope you find amusing:

My car's been driving crooked down the lane
It's been too long since it's been right

The machanic's face he wears so grim
My request he cannot fulfill-
The alignment I seek
He cannot meet as one wheel 
has gone astray.

Home I go crooked and slow
As my mechanic waves in my mirror

Mike
